src/main/java/github/grace000/erversay/Response/ResponseWriter.java

```diff
@@ -0,0 +1,26 @@
+package github.grace000.erversay.Response;
+
+import java.io.*;
+
+import static github.grace000.erversay.Constants.HTTPLines.*;
+
+public class ResponseWriter {
+    public void write(OutputStream outputStream, Response response) {
+        try {
+            byte[] status = (DEFAULT_VERSION + SP + response.status + CRLF).getBytes();
+            byte[] headers = (response.headers).getBytes();
+
+            outputStream.write(status);
+            outputStream.write(headers);
+            outputStream.write((DOUBLE_LINE_FEED).getBytes());
+            outputStream.write(response.body);
+
+            System.out.println("WRITER: MESSAGE SENT");
```

Comment:
Do you want the logging still in here?

Reply:
I don't need it. It helps me see the extra pings coming through, but I'll delete.